dynamic_programming/hamiltons-cycle.cpp

```diff
@@ -1,9 +1,15 @@
 /**
- * @author vakhokoto
+ * @file
+ * @brief The implementation of hamilton's cycle dynamic solution for vertices number less than 20.
```

Comment:
```suggestion
 * @brief The implementation of [hamilton's cycle](https://en.wikipedia.org/wiki/Hamiltonian_path_problem) dynamic solution for vertices number less than 20.
```
Let us add some links so that viewers can learn more about the problem itself.